src/main/scala/nl.knaw.dans.easy.multideposit/package.scala

```diff
@@ -114,8 +114,8 @@ package object multideposit {
      */
     @throws[IOException]("in case of an I/O error")
     def writeXml(elem: Elem, encoding: Charset = encoding): Unit = {
-      Files.createDirectories(file.getParent)
-      XML.save(file.toString, elem, encoding.toString, xmlDecl = true)
+      Files.createDirectories(path.getParent)
+      XML.save(path.toString, XML.loadString(new PrettyPrinter(160, 2).format(Utility.trim(elem))), encoding.toString, xmlDecl = true)
```

Comment:
I've been looking in the API and there is only an `XML.save` with a `String` to point to the file and a `Elem` pointing to the xml to be written. In this (and following) method(s) the header is added and the xml is written to the file. Talking about bad API design 😛.
As you noticed yesterday the formatting was messed up without doing the `PrettyPrinter.format` first, so this is the solution I came up with. I want to stay as close to library calls as possible here, rather than copy-pasting stuff and modifying it for our own purposes.